content/swiftui/concepts/color/color.md

```diff
@@ -0,0 +1,73 @@
+---
+Title: 'Color'
+Description: 'Creates a color instance.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI ViewModifiers'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+**`Color`** creates a color instance. `Color`s can be used as arguments, for example to fill backgrounds and shapes or to apply color to text. They can also be used directly as [views](https://www.codecademy.com/resources/docs/swiftui/views), creating a rectangle area filled with their color, that adapts to fill the space available.
+
+## Syntax
+
+`Color` instances can be declared using dot notation or bracket notation:
+
+```pseudo
+
+let someColor = Color.color property
+
+let someColor = Color(.color property or variable and/or color component values)
+
+```
+
+Here are some of the standard color properties that are available. Many of the color properties will change depending on their context, for example if light or dark mode is set.
+
+- `.red` (context dependent)
+- `.orange` (context dependent)
+- `.green` (context dependent)
+- `.white`
+- `.black`
+- `.primary` (context dependent)
+- `.secondary` (context dependent)
+
+The SwiftUI developer documentation specifies how to [create colors from component values](https://developer.apple.com/documentation/swiftui/color).
```

Comment:
Is it possible to include the component information here rather than linking out to off-site documentation?

Reply:
@SSwiniarski I added in the component information and an example for them.